cmd/libvirt-actuator/README.md

```diff
@@ -2,28 +2,114 @@
 
 The command allows to directly interact with the libvirt actuator.
 
+## How to prepare environment
+
+For running the libvirt actuator with Fedora 28 Cloud images.
+
+1. Provision instance in `packet.net`:
+   ```sh
+   $ ./hack/packet-provision.sh install
+   ```
+   and get the IP address (assuming it is `147.75.96.139`).
+
+1. Update ``~/.ssh/config`` to contain:
+   ```
+   Host libvirtactuator 147.75.96.139
+   Hostname 147.75.96.139
+   ControlPersist 10m
+   ControlMaster auto
+   ControlPath /tmp/%r@%h:%p
+   User root
+   StrictHostKeyChecking no
+   PasswordAuthentication no
+   UserKnownHostsFile ~/.ssh/aws_known_hosts
+   IdentityFile /tmp/packet_id_rsa
+   IdentitiesOnly yes
+   Compression yes
+   ```
+
+1. Run [init script](resources/init.sh) to:
+   - update qemu-kvm to at least 2.4
+   - create default volume storage
+   - create cloud init volume
+   - create `fedora_base` volume
+
+   ```sh
+   $ bash cmd/libvirt-actuator/resources/init.sh
+   ```
+
+   In case the default [cloud-init](resources/user-data) is not sufficient,
+   you can generate new `cloud-init.iso` with the following steps:
+
+   1. Update content of [meta-data](resources/meta-data) file (the filename must be `meta-data`)
+   1. Update content of [user-data](resources/user-data) (the filename must be `user-data`)
+   1. Generate new iso with the cloud init by running:
+      ```sh
+      $ genisoimage -output cloud-init.iso -volid cidata -joliet -r user-data meta-data
+      ```
+   1. Upload the iso to the instance in case you already run the `init.sh` script:
+      ```
+      $ scp cloud-init.iso libvirtactuator:/var/lib/libvirt/images/cloud-init.iso
```

Comment:
Given that we have user/admin rights to the pool you can use virsh to do this:

    $ virsh vol-upload --pool default --vol cloud-init.iso --file /path/to/cloud-init.iso

Reply:
Even better, thanks :)